hyrule_field.logic

```diff
@@ -127,16 +127,95 @@ area HyruleField:
     MiddleEast -> Center: (item RocsCape)
     Center -> MinishCenter: (item PegasusBoots | flag CanTurnMinish)
     MinishCenter -> Center: (flag CanTurnMinish)
-    # Assume this entrance/exit won't be randomized if no fusions
-    Center <-> PondFusion: (flag PondFusion)
+    Center -> PondFusion: (flag PondFusion)
+    PondFusion -> Center
     Center -> NorthLedge: (item RocsCape | item PacciCane )
     NorthLedge -> Center
     NorthLedge -> MiddleEast
     NorthLedge <-> MinishCrack: (flag CanTurnMinish)
     NorthLedge -> CenterLedge: (flag ObscureSpots & item MoleMitts)
     NorthFreestanding -> NorthDiggingCave: (item MoleMitts)
     NorthDiggingCave -> NorthFreestanding
-  room North
+  room North:
+    node Main:
+      door South
+      door North
+      enemy OctorcsAndCrows
+    node NWTingleTree:
+      door NWTingleTree
```

Comment:
Doing it this way implies you can go back through the entrance if the tree is closed (you'd be able to leave, but no reenter, right?)

Reply:
Ah ?  I thought it wouldn't because of those edges
``Main -> NWTingleTree: flag NWTingleTree``
  ``NWTingleTree -> Main``

Reply:
That just means you can't get to the entrance from anywhere else without the flag being set - if the exit and entrance are on the same node (whenever you use `door`) you are implied to be able to get back freely.

Reply:
Oh ok, so if I get this right, I use door and the same nodz only if it's the access to the door, and exit/entrance and separeted nodes when it's the door itself ?